charts/neuvector/v1.6.8/templates/crd.yaml

```diff
@@ -0,0 +1,428 @@
+                      type: object
+                  type: object
+                type: array
+              file:
+                items:
+                  properties:
+                    app:
+                      items:
+                        type: string
+                      type: array
+                    behavior:
+                      type: string
+                    filter:
+                      type: string
+                    recursive:
+                      type: boolean
+                  type: object
+                type: array
+              ingress:
+                items:
+                  properties:
+                    action:
+                      type: string
+                    applications:
+                      items:
+                        type: string
+                      type: array
+                    name:
+                      type: string
+                    ports:
+                      type: string
+                    selector:
+                      properties:
+                        criteria:
+                          items:
+                            properties:
+                              key:
+                                type: string
+                              op:
+                                type: string
+                              value:
+                                type: string
+                            type: object
+                          type: array
+                        name:
+                          type: string
+                      type: object
+                  type: object
+                type: array
+              process:
+                items:
+                  properties:
+                    action:
+                      type: string
+                    name:
+                      type: string
+                    path:
+                      type: string
+                  type: object
+                type: array
+              target:
+                properties:
+                  policymode:
+                    type: string
+                  selector:
+                    properties:
+                      criteria:
+                        items:
+                          properties:
+                            key:
+                              type: string
+                            op:
+                              type: string
+                            value:
+                              type: string
+                          type: object
+                        type: array
+                      name:
+                        type: string
+                    type: object
+                type: object
+            required:
+            - target
+            type: object
+        type: object
+{{- end }}
+---
+{{- if (semverCompare ">=1.19-0" (substr 1 -1 .Capabilities.KubeVersion.GitVersion)) }}
+apiVersion: apiextensions.k8s.io/v1
+{{- else }}
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
this one as well

Reply:
We want to apply CRD differently depending on the k8s version. Our previous version rancher catalog 1.6.0 has also the same CRD, but it was approved before. Curious to know what is different now. 

By chance, do you run helm lint against a k8s version?